src/sinks/gcp/stackdriver/metrics/mod.rs

```diff
@@ -0,0 +1,20 @@
+// TODO: In order to correctly assert component specification compliance, we would have to do some more advanced mocking
+// off the endpoint, which would include also providing a mock OAuth2 endpoint to allow for generating a token from the
+// mocked credentials. Let this TODO serve as a placeholder for doing that in the future.
+
+use crate::sinks::prelude::*;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏is all that we need `SinkBatchSettings` in this file? Seems like could import that one directly.

Reply:
They should probably be in `config.rs` really.